app/backend/src/database/Database.ts

```diff
@@ -22,7 +22,14 @@ export class Database {
           path.join(__dirname, "migrations"),
         ),
       },
-      pool: config.connectionPoolSize,
+      pool: {
+        ...config.connectionPoolSize,
+        afterCreate: function (connection: any, callback: any) {
+          connection.query("SET TIMEZONE TO 'UTC';", function (err: any) {
```

Comment:
This sets every new connection time zone to UTC. Otherwise node-postgres (invoked by knex) used local server time zone for database operations (like `DATE_TRUNC`) which made it very cumbersome to work with.

Reply:
Suggestion taken from here: https://github.com/knex/knex/issues/97